app/src/main/kotlin/com/hippo/ehviewer/spider/SpiderDen.kt

```diff
@@ -233,24 +232,16 @@ class SpiderDen(mGalleryInfo: GalleryInfo) {
     }
 }
 
-private val COMPAT_IMAGE_EXTENSIONS = SUPPORT_IMAGE_EXTENSIONS + ".jpeg"
-
 /**
  * @param extension with dot
  */
 fun perFilename(index: Int, extension: String?): String {
     return String.format(Locale.US, "%08d%s", index + 1, extension)
 }
 
-/**
- * @param extension with dot
- */
-private fun fixExtension(extension: String): String {
-    return extension.takeIf { SUPPORT_IMAGE_EXTENSIONS.contains(it) } ?: SUPPORT_IMAGE_EXTENSIONS[0]
-}
-
 private fun findImageFile(dir: UniFile, index: Int): UniFile? {
-    return COMPAT_IMAGE_EXTENSIONS.firstNotNullOfOrNull { dir.findFile(perFilename(index, it)) }
+    val head = String.format(Locale.US, "%08d", index + 1)
```

Comment:
nit: `perFilename(index, ".")` might be better here.

Reply:
Done